README.md

```diff
@@ -111,10 +102,7 @@ Alternately, you can target a more specific sub-package instead of `all`.
 Testing and Coverage
 --------------------
 
-Testing with coverage tracking is currently only available via
-`stack`, as `cabal new-*` [does not yet support coverage](https://github.com/haskell/cabal/issues/5213).
-Use `scripts/stack-test-coverage.sh` to generate a coverage
-report for all test suites.
+Testing with coverage tracking is done via `cabal test --enable-coverage ...`
```

Comment:
Make sure to mention to mention the workaround for https://github.com/GaloisInc/crucible/issues/884 here.

Reply:
+1